Sourcery/Parsing/Composer.swift

```diff
@@ -98,6 +98,30 @@ struct Composer {
         }
 
         let lookupName = typeName.actualTypeName ?? typeName
+        
+        if lookupName.generic == nil && lookupName.isGeneric {
```

Comment:
Good thing my code is being checked now :) Never worked with code like this before (NSCoding/NSCoder). Working on my own projects is a lot easier being familiar with the code base.